packages/roosterjs-react-editor/lib/components/LeanRooster.tsx

```diff
@@ -113,6 +111,36 @@ export default class LeanRooster extends React.Component<LeanRoosterProps, {}> {
         }
     }
 
+    public componentWillReceiveProps(nextProps: LeanRoosterProps, nextState: {}) {
+        const div = this._contentDiv;
+        if (!div) {
+            return;
+        }
+
+        const { className, readonly, isRtl, placeholder } = nextProps;
+        
+        if (className !== this.props.className) {
+            div.setAttribute("class", this._getClassName(nextProps));
```

Comment:
This will remove any other existing class names on this div. Is that acceptable?

Reply:
Never mind, noticed you call _getClassName. Ignore. :)